datamodel/src/main/java/com/hartwig/serve/datamodel/EvidenceDirection.java

```diff
@@ -1,24 +1,31 @@
 package com.hartwig.serve.datamodel;
 
 public enum EvidenceDirection {
-    RESPONSIVE(true, false, true),
-    PREDICTED_RESPONSIVE(true, false, false),
-    NO_BENEFIT(false, false, true),
-    RESISTANT(false, true, true),
-    PREDICTED_RESISTANT(false, true, false);
+    RESPONSIVE(true, false, false, true),
+    PREDICTED_RESPONSIVE(true, false, false, false),
+    DECREASED_RESPONSE(false, false, false, true),
+    NO_BENEFIT(false, true, false, true),
+    RESISTANT(false, true, true, true),
+    PREDICTED_RESISTANT(false, true, true, false);
 
-    private final boolean isResponsive;
+    private final boolean hasPositiveResponse;
+    private final boolean hasNoBenefit;
     private final boolean isResistant;
     private final boolean isCertain;
 
-    EvidenceDirection(final boolean isResponsive, final boolean isResistant, final boolean isCertain) {
-        this.isResponsive = isResponsive;
+    EvidenceDirection(final boolean hasPositiveResponse, final boolean hasNoBenefit, final boolean isResistant, final boolean isCertain) {
+        this.hasPositiveResponse = hasPositiveResponse;
+        this.hasNoBenefit = hasNoBenefit;
         this.isResistant = isResistant;
         this.isCertain = isCertain;
     }
 
-    public boolean isResponsive() {
-        return isResponsive;
```

Comment:
You are not allowed to remove a function from a datamodel object since this will break backwards compatibility.

What you need to do is to keep this function, have it return `hasPositiveResponse` (assuming that's correct at least!), and annotate it with `@Deprecated`, so that clients know they should not use this function anymore.